### PR TITLE
Feature a

### PR DESCRIPTION
**Pull request by phat-ng-eroad**
Hi @Nana3012 ,
I just changed the title to "The band NZ". Please review it

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>The band</title>
+    <title>The band NZ</title>
```
Comment (Nana3012) [inline index.html:8]: all gud

File: hello.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+dasdasd
+a
+da
+sd
+a
+d
```
Comment (Nana3012) [inline hello.txt]: please remove the unncessary file

**Review by Nana3012 — COMMENTED** [review]
I just left some comments for u

**Comment by phat-ng-eroad** [comment]
hi @Nana3012 . 
Please re-review this PR

**Review by Nana3012 — APPROVED** [review]

**Pull request merged**